docs/resources/integration_aws_event_bridge.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_aws_event_bridge Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a datadogintegrationawseventbridge resource. This can be used to create and manage Event Sources for each Datadog integrated AWS account.
+---
+
+# datadog_integration_aws_event_bridge (Resource)
+
+Provides a datadog_integration_aws_event_bridge resource. This can be used to create and manage Event Sources for each Datadog integrated AWS account.
```

Comment:
```suggestion
Provides a datadog_integration_aws_event_bridge resource. This can be used to create and manage Event Sources for each Datadog-integrated AWS account.
```